caches.go

```diff
@@ -92,6 +93,36 @@ func (rcc *clientRegionCache) clientDown(c hrpc.RegionClient) map[hrpc.RegionInf
 	return downregions
 }
 
+// Collects information about the clientRegion cache and stores them in two maps to reduce
+// duplication of data. We do this in one function to avoid running the iterations twice
```

Comment:
Instead of "store them in two maps" mention that it appends the data into the given maps